app/views/categories/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<nav class="header-categories d-flex flex-center">
+  <%= link_to  destroy_user_session_path, class: "icons" do%>
+    <i class="fa-solid fa-toggle-on"></i>
+  <% end %>
+    <h1>CATEGORIES</h1>
+</nav>
+
```

Comment:
- [ ] Kindly use the icon for `log out` or text for it to have a meaningful or descriptive message on our project.

![image](https://user-images.githubusercontent.com/55635977/154809590-60490500-56bc-4cd3-8dc1-32472bb3ad30.png)

Reply:
Yes sure, I'm going to change it thanks!!